psi4/driver/procrouting/sapt/sapt_proc.py

```diff
@@ -34,77 +34,131 @@
     core.print_out("\n")
-    core.print_out("         ---------------------------------------------------------\n")
+    core.print_out(
+        "         ---------------------------------------------------------\n"
+    )
     core.print_out("         " + "SAPT(DFT) Procedure".center(58) + "\n")
     core.print_out("\n")
     core.print_out("         " + "by Daniel G. A. Smith".center(58) + "\n")
-    core.print_out("         ---------------------------------------------------------\n")
+    core.print_out(
+        "         ---------------------------------------------------------\n"
+    )
     core.print_out("\n")
 
     # core.print_out("  !!!  WARNING:  SAPT(DFT) capability is in beta. Please use with caution. !!!\n\n")
-    core.print_out("Warning! The default value of SAPT_DFT_EXCH_DISP_SCALE_SCHEME has changed from DISP to FIXED. Please be careful comparing results with earlier versions. \n\n")
+    core.print_out(
+        "Warning! The default value of SAPT_DFT_EXCH_DISP_SCALE_SCHEME has changed from DISP to FIXED. Please be careful comparing results with earlier versions. \n\n"
+    )
 
     core.print_out("  ==> Algorithm <==\n\n")
-    core.print_out("   SAPT DFT Functional     %12s\n" % str(sapt_dft_functional))
+    core.print_out("   SAPT DFT Functional     %12s\n" %
+                   str(sapt_dft_functional))
     core.print_out("   Monomer A GRAC Shift    %12.6f\n" % mon_a_shift)
     core.print_out("   Monomer B GRAC Shift    %12.6f\n" % mon_b_shift)
+    # fmt: off
     core.print_out("   Delta HF                %12s\n" % ("True" if do_delta_hf else "False"))
     core.print_out("   JK Algorithm            %12s\n" % core.get_global_option("SCF_TYPE"))
+    # fmt: on
     core.print_out("\n")
     core.print_out("   Required computations:\n")
-    if (do_delta_hf):
-        core.print_out("     HF  (Dimer)\n")
-        core.print_out("     HF  (Monomer A)\n")
-        core.print_out("     HF  (Monomer B)\n")
-    if (do_dft):
-        core.print_out("     DFT (Monomer A)\n")
-        core.print_out("     DFT (Monomer B)\n")
+    if do_delta_hf:
+        core.print_out("     HF   (Dimer)\n")
+        core.print_out("     HF   (Monomer A)\n")
+        core.print_out("     HF   (Monomer B)\n")
+    if do_dft:
+        core.print_out("     DFT  (Monomer A)\n")
+        core.print_out("     DFT  (Monomer B)\n")
+    if do_mon_grac_shift_A:
+        core.print_out("     GRAC (Monomer A)\n")
+        mon_a_shift = compute_GRAC_shift(
+            monomerA_mon_only_bf,
+            grac_compute,
+            "Monomer A",
+        )
+    if do_mon_grac_shift_B:
+        core.print_out("     GRAC (Monomer B)\n")
+        mon_b_shift = compute_GRAC_shift(
+            monomerB_mon_only_bf,
+            grac_compute,
+            "Monomer B",
+        )
+
+    core.set_variable("SAPT DFT GRAC SHIFT A", mon_a_shift)  # P::e SAPT
+    core.set_variable("SAPT DFT GRAC SHIFT B", mon_b_shift)  # P::e SAPT
```

Comment:
don't forget the glossary.rst entries

Reply:
yes, those two vars, but they'll look like the below. And put them somewhere in the SAPT area since the file doesn't alphabetize.

```
.. psivar:: SAPT DFT GRAC SHIFT A
   SAPT DFT GRAC SHIFT B

   The gradient-corrected ... [E_h] ...
```